packages/scandipwa/src/component/MyAccountDashboard/MyAccountDashboard.component.tsx

```diff
@@ -118,10 +122,12 @@ export class MyAccountDashboardComponent extends PureComponent<MyAccountDashboar
                 <div block="MyAccountDashboard" elem="BlockTitle">
                     <span>{ __('Account Information') }</span>
                 </div>
-                <MyAccountCustomerTable
-                  customer={ customer }
-                  title={ __('My profile') }
-                />
+                <Suspense fallback={ <div block="MyAccountDashboard" elem="MyAccountCustomerTableFallback" /> }>
```

Comment:
ideally, we should have a separate rendering method for the fallback. it will make the customizations easier.